scripts/check-size.sh

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env bash
 #
+
 # While Solana doesn't support resizing programs, the oracle binary needs to be smaller than 81760 bytes
+# (The available space for the oracle program on pythnet is 88429 and mainnet is 81760)
 ORACLE_SIZE=$(wc -c ./target/deploy/pyth_oracle.so | awk '{print $1}')
-if [ $ORACLE_SIZE -lt 81760 ]
+if [ $ORACLE_SIZE -lt 88429 ]
```

Comment:
I think we the c flags we can stay under this with the 3 upcoming changes : twap + resize + message-buffer